train.py

```diff
@@ -170,6 +170,18 @@ def validate_directories(args):
     }
 
 
+def check_sample_size(dilations, sample_size):
+    '''Check that the sample size is greater than the receptive field size and
```

Comment:
It should be double quotes, but this is not your fault -- I'll send a refactor PR when this is merged changing all of them.